draft-schwartz-httpbis-dns-alt-svc.md

```diff
@@ -180,7 +180,7 @@ records, the server operator MUST set the overall TTL to the minimum
 of the "max age" values (following Section 5.2 of {{!RFC2181}}).
 Each RR can contain multiple alt-values separated by commas, as described
 in Section 3 of {{!AltSvc}}.  As specified there, "the order of values
-relects the server's preference".
+reflects the server's preference".
```

Comment:
This draft is obsolete, so I don't think we should make any further changes to it.